apps/trackers/tests/test_integration.py

```diff
@@ -26,153 +25,146 @@
-#
-# @pytest.mark.vcr
-# def test_tracker_update_bugzilla(self):
-#     """
-#     test basic Bugzilla tracker update
-#     """
-#     # 1) define all the context
-#     ps_module = PsModuleFactory(
-#         bts_name="bugzilla",
-#         bts_key="Red Hat Certification Program",
-#         bts_groups={"public": ["devel"]},
-#         default_component="redhat-certification",
-#         name="rhcertification-6",
-#     )
-#     ps_update_stream = PsUpdateStreamFactory(
-#         name="rhcertification-6",
-#         ps_module=ps_module,
-#         version="1.0",
-#     )
-#     flaw = FlawFactory(
-#         bz_id="2013494",
-#         embargoed=False,
-#         impact=Impact.IMPORTANT,
-#     )
-#     affect = AffectFactory(
-#         flaw=flaw,
-#         ps_module=ps_module.name,
-#         ps_component="openssl",
-#         affectedness=Affect.AffectAffectedness.AFFECTED,
-#         resolution=Affect.AffectResolution.DELEGATED,
-#     )
-
-#     # 2) define a tracker model instance
-#     #    according an exising Bugzilla tracker
-#     tracker_id = "2018651"
-#     updated_dt = "2023-10-11T15:48:03Z"
-#     tracker = TrackerFactory(
-#         affects=[affect],
-#         bz_id=tracker_id,
-#         embargoed=flaw.embargoed,
-#         ps_update_stream=ps_update_stream.name,
-#         type=Tracker.TrackerType.BUGZILLA,
-#         meta_attr={"blocks": ["2013494"], "updated_dt": updated_dt},
-#         updated_dt=timezone.datetime.strptime(updated_dt, BZ_DT_FMT),
-#     )
-#     assert tracker.bz_id == tracker_id
-
-#     # 3) update tracker in OSIDB and Bugzilla
-#     ts = TrackerSaver(tracker, bz_api_key="SECRET")
-#     updated_tracker = ts.save()
-#     assert updated_tracker.bz_id == tracker_id
-
-#     # 4) load tracker from Bugzilla
-#     #    this must be done through flaw collector
-#     #    because that one is responsible for linking
-#     fc = FlawCollector()
-#     fc.sync_flaw(flaw.bz_id)
-
-#     # 5) get the newly loaded tracker from the DB
-#     loaded_tracker = Tracker.objects.get(external_system_id=tracker_id)
-
-#     # 6) check the correct result of the update and loading
-#     assert loaded_tracker.bz_id == tracker_id
-#     assert not loaded_tracker.embargoed
-#     assert loaded_tracker.type == Tracker.TrackerType.BUGZILLA
-#     assert loaded_tracker.ps_update_stream == "rhcertification-6"
-#     assert loaded_tracker.status == "NEW"
-#     assert not loaded_tracker.resolution
-#     assert loaded_tracker.affects.count() == 1
-#     assert loaded_tracker.affects.first() == affect
-#     assert not loaded_tracker._alerts
-
-#     # 7) check that the update actually happened
-#     assert "updated_dt" in loaded_tracker.meta_attr
-#     assert updated_dt != loaded_tracker.meta_attr["updated_dt"]
+class TestTrackerSaver:
+    @pytest.mark.vcr
+    def test_tracker_create_bugzilla(self):
+        """
+        test basic Bugzilla tracker creation
+        """
+        # 1) define all the context
```

Comment:
Now I see why you commented this code in "remove tracker sync from flaw sync".

I would rather remove there and add here for cleaner/easier "reverse cherry-picking" if needed in future but I believe this is fine, you can freely ignore both this and the past comment about this test.